JASP-Engine/JASP/R/ttestindependentsamples.R

```diff
@@ -623,18 +623,16 @@ TTestIndependentSamples <- function(dataset = NULL, options, perform = "run",
 
 				row.footnotes <- NULL
 				error <- FALSE
-
-				if (length(data) < 3) {
-					err <- .generateErrorMessage(type='levene', observations.amount='< 3', variables=variable, grouping=factor)
-					foot.index <- .addFootnote(footnotes, err)
-					row.footnotes <- list(W = list(foot.index), p = list(foot.index))
-					error <- TRUE
-
-				} else if (length(data) > 5000) {
-					err <- .generateErrorMessage(type='levene', observations.amount='> 5000', variables=variable, grouping=factor)
-					foot.index <- .addFootnote(footnotes, err)
-					row.footnotes <- list(W = list(foot.index), p = list(foot.index))
-					error <- TRUE
+				
+				errors <- .hasErrors(dataset, perform, message = 'short', type = c('observations', 'variance', 'infinity'),
```

Comment:
Also, in your .hasErrors() calls in the independent samples t-test you do not specify the grouping variable.
One group might have 20 observations in one group and in the other zero.